joomla/joomla-cms
# [4.0] Cleanup "to be removed" list in script.php

Pull Request for Issue #16704 .

### Summary of Changes
The `deleteUnexistingFiles` has some files and a folder in its list which were removed in earlier Joomla versions, but added back in 4.0. Obviously we should not remove them anymore on an update :smile: 

This PR also deletes the Jlayout override `administrator/components/com_languages/layouts/joomla/searchtools/default.php`. This file got removed with https://github.com/joomla/joomla-cms/pull/10090 but magically appears back in J4. I can only assume this is some merge error. Deleting the file seems to make no difference in the "Extensions -> Language(s) -> Installed" view.

### Testing Instructions
Click the database fix button and verify the only file that is being deleted is the weblinks package one.


### Expected result
Only pkg_weblinks.xml should be removed


### Actual result
These files are removed as well
* administrator/components/com_languages/layouts/joomla/searchtools/default.php
* administrator/components/com_media/models/forms/file.xml
* media/com_banners/js/admin-banner-edit.js
* media/com_finder/js/finder.js
* media/editors/codemirror/css/codemirror.css


### Documentation Changes Required
None
